google/cloud/storage/bucket_metadata.h

```diff
@@ -212,18 +212,23 @@ struct BucketIamConfiguration {
   ///     `uniform_bucket_level_access`
   absl::optional<BucketPolicyOnly> bucket_policy_only;
   absl::optional<UniformBucketLevelAccess> uniform_bucket_level_access;
+  absl::optional<std::string> public_access_prevention;
 };
 
 //@{
 /// @name Comparison operators for BucketIamConfiguration.
 inline bool operator==(BucketIamConfiguration const& lhs,
                        BucketIamConfiguration const& rhs) {
-  return lhs.uniform_bucket_level_access == rhs.uniform_bucket_level_access;
+  return lhs.uniform_bucket_level_access == rhs.uniform_bucket_level_access &&
+         lhs.public_access_prevention == rhs.public_access_prevention;
 }
 
 inline bool operator<(BucketIamConfiguration const& lhs,
                       BucketIamConfiguration const& rhs) {
-  return lhs.uniform_bucket_level_access < rhs.uniform_bucket_level_access;
+  return std::tie(lhs.uniform_bucket_level_access,
```

Comment:
Opt: If we use `tie` for `operator<`, perhaps we should also use it for `operator==`.

Reply:
Done